scripts/gis-material-update/README.md

```diff
@@ -326,21 +326,20 @@ Output files (names configured in `config.yaml`)
 
 ### `liikennevaylat`
 
+Fill out following secrets in gis-material-update/.env:
```

Comment:
These are needed above in "Run data fetch" section (under line 108). One cannot run `docker-compose run --rm gis-fetch liikennevaylat` without these envs.

Reply:
Now changed location of secret documentation